gabbi/handlers/jsonhandler.py

```diff
@@ -94,9 +94,11 @@ def action(self, test, path, value=None):
 
         expected = test.replace_template(value)
         # If expected is a string, check to see if it is a regex.
-        if (hasattr(expected, 'startswith') and expected.startswith('/')
-                and expected.endswith('/')):
-            expected = expected.strip('/').rstrip('/')
+        if (isinstance(expected, six.string_types)
```

Comment:
The reason I used `hasattr` is because I have some kind of bias against `isinstance` that I should probably just get over.

Reply:
Maybe reading https://docs.python.org/3/library/abc.html#abc.ABCMeta.__subclasshook__ will help you get over it ;)